[2024-09-30T14:08:08.338+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-30T14:08:08.345+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mssql_to_postgres_sqlalchemy.fact_run.fact_2 manual__2024-09-30T14:05:00.391775+00:00 [queued]>
[2024-09-30T14:08:08.349+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mssql_to_postgres_sqlalchemy.fact_run.fact_2 manual__2024-09-30T14:05:00.391775+00:00 [queued]>
[2024-09-30T14:08:08.350+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-30T14:08:08.358+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): fact_run.fact_2> on 2024-09-30 14:05:00.391775+00:00
[2024-09-30T14:08:08.362+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=433) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-30T14:08:08.363+0000] {standard_task_runner.py:72} INFO - Started process 438 to run task
[2024-09-30T14:08:08.364+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'mssql_to_postgres_sqlalchemy', 'fact_run.fact_2', 'manual__2024-09-30T14:05:00.391775+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/load.py', '--cfg-path', '/tmp/tmpk2kvouox']
[2024-09-30T14:08:08.367+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask fact_run.fact_2
[2024-09-30T14:08:08.391+0000] {task_command.py:467} INFO - Running <TaskInstance: mssql_to_postgres_sqlalchemy.fact_run.fact_2 manual__2024-09-30T14:05:00.391775+00:00 [running]> on host 51a7ea638889
[2024-09-30T14:08:08.435+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mssql_to_postgres_sqlalchemy' AIRFLOW_CTX_TASK_ID='fact_run.fact_2' AIRFLOW_CTX_EXECUTION_DATE='2024-09-30T14:05:00.391775+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-30T14:05:00.391775+00:00'
[2024-09-30T14:08:08.436+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-30T14:08:08.457+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-30T14:08:08.464+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-30T14:08:08.465+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=mssql_to_postgres_sqlalchemy, task_id=fact_run.fact_2, run_id=manual__2024-09-30T14:05:00.391775+00:00, execution_date=20240930T140500, start_date=20240930T140808, end_date=20240930T140808
[2024-09-30T14:08:08.503+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-30T14:08:08.509+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
